.github/workflows/integration_test.yaml

```diff
@@ -0,0 +1,26 @@
+name: Integration tests
+
+on:
+  pull_request:
+
+jobs:
+  integration-test-microk8s:
+    name: Integration tests (microk8s)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup operator environment
+        uses: charmed-kubernetes/actions-operator@main
+        with:
+          juju-channel: 3.1/stable
+          provider: microk8s
+          channel: 1.25-strict/stable
```

Comment:
Is it required to run `1.25` or can it be bumped to current (`1.28`)

Reply:
For consistency with our other charms I'm leaving at 1.25 but will look into bumping this in future.